agent-nodejs-10/Dockerfile.rhel8

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/4.8:jenkins-agent-base
+FROM registry.ci.openshift.org/ocp/4.12:jenkins-agent-base
```

Comment:
it should be 4.11 for jenkins-agent-base

Reply:
updated in current push